iroh-net/src/magicsock.rs

```diff
@@ -484,6 +485,7 @@ impl Inner {
         Ok(sock)
     }
 
+    // NOTE: Receiving on a [`Self::closed`] socket will return [`Poll::Pending`] undefinitely.
```

Comment:
This should be mentioned in the doc comment of the `MagicSock` itself, or at least of it's `AsyncUdpSocket` impl.

Reply:
done in c8fa0de